INSTALL.md

```diff
@@ -284,81 +284,84 @@ If there are other files that you have customized, then you will also need to tr
-
-**Note:**
-1. Make sure that your XAMPP control panel has Apache and MySQL turned on
-2. Apache needs to be on port 80, 443
-3. mySQL needs to be on port 3306
-
-#### Step 2
-Leave default as the "Site ID:" and press continue.
-
-![First Step](./Documentation/1_Installing/images/windows_installation/Step_1.png)
-
-Make sure that there are no undefined index errors, if so make sure that you changed the php.ini file, or have the correct version of XAMPP.
-
-![Second Step](./Documentation/1_Installing/images/windows_installation/Step_2.png)
-
-Then after the second step, continue and leave the option "Have setup create the database" and press continue
-
-![Third Step](./Documentation/1_Installing/images/windows_installation/Step_3.png)
-
-For the fourth step, enter a "Password" and "Initial User Password" you are free to change, the "Initial User" to your own username, but for convenience you can also leave it as admin. Then press continue
-
-![Fourth Step](./Documentation/1_Installing/images/windows_installation/Step_4.png)
-
-Then after the fourth step, you can press continue through the others as long as the steps above were followed with clarity, the rest should have no errors, and each remaining page can be continued without change.
-
-![Fifth Step](./Documentation/1_Installing/images/windows_installation/Step_5.png)
-
-![Sixth Step](./Documentation/1_Installing/images/windows_installation/Step_6.png)
-
-![Seventh Step](./Documentation/1_Installing/images/windows_installation/Step_7.png)
-
-![Eigth Step](./Documentation/1_Installing/images/windows_installation/Step_8.png)
-
-![Ninth Step](./Documentation/1_Installing/images/windows_installation/Step_9.png)
-
-![Tenth Step](./Documentation/1_Installing/images/windows_installation/Step_10.png)
-
-**Yay, you have successfull setup LibreEHR!**
+ To run LibreEHR on Windows, [XAMPP](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) is needed with a compatible version
+ 
+ **Note:** 
+ 
+ 1. Must have php [7.0](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/_) or [5.6](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/) ([5.6.30](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) recommended)
+ 
+ 2. php 7.1x is not currently supported
+ 
+ Clone the LibreEHR [repository](https://github.com/LibreHealthIO/LibreEHR) into your console eg. [GitBash](https://git-for-windows.github.io/), [Cmder](http://cmder.net/)
+ 
+ After cloning LibreEHR, the `php.ini` file is located in `xampp\php\` directory.
+ 
+ Open this file with your text editor eg. [Subl](https://www.sublimetext.com/3) or Notepad (which is already installed on your computer).
+ 
+ There will be 4 php files located in `xampp\php`, but the one needed is the one that's type is "Configuration Settings" that  change the following in the php.ini file:
+ 
+**Note: You can use Crtl+F in your text editor to find the specific lines that you need to change accordingly.**
```

Comment:
Is Crtl+f the same across all editors? Would it be better to say 'You can use Search (ctrl+f in my editor)